src/traits.rs

```diff
@@ -403,6 +403,23 @@ pub trait ProcessExt: Debug {
     /// ```
     fn user_id(&self) -> Option<&Uid>;
 
+    /// Returns the ID of the effective owner user of this process or `None` if this information
```

Comment:
Would be nice to have a small sentence to explain the difference between effective and real (like the one you already wrote yesterday :wink: ).